client/src/components/EpisodeCard.js

```diff
@@ -34,23 +55,25 @@ const Wrapper = styled.li`
 
 const Heading = styled.div`
```

Comment:
Hm, I wouldn't call it `Heading` as I would expect a styled headline element. `HeadingContainer` maybe?